src/sagas/networkSettings.js

```diff
@@ -289,6 +289,8 @@ export function* persistNetworkSettings(action) {
     return;
   }
 
+  // Remove registered tokens
+  STORE.removeItem(REGISTERED_TOKENS_KEY);
```

Comment:
Does it remove from the wallet storage as well, in the `wallet.storage`?

Reply:
This one: https://github.com/HathorNetwork/internal-issues/issues/285

Reply:
Yes, it does.

Reply:
I refactored it.